pyiron_contrib/atomistics/atomistics/job/trainingcontainer.py

```diff
@@ -172,9 +174,26 @@ def run_if_interactive(self):
 
     def to_hdf(self, hdf=None, group_name=None):
         super().to_hdf(hdf=hdf, group_name=group_name)
-        with catch_warnings():
-            self._table.to_hdf(self.project_hdf5.file_name, self.name + "/output/structure_table")
+        self._container.to_hdf(self.project_hdf5, "structures")
```

Comment:
Yeah, this is beautiful. This is exactly what overrides of `to_hdf` should look like 😄